projectDocs/translating/crowdin.md

```diff
@@ -0,0 +1,153 @@
+# Translating using Crowdin
+
+Crowdin is used to translate the main NVDA interface.
+NVDA's Crowdin project: <https://crowdin.com/project/nvda>.
+
+This document covers setting up a Crowdin account, connecting it with PoEdit, and translating the main interface using Crowdin and PoEdit.
+
+## Setup
+
+### Create Crowdin account
+
+1. Create a [Crowdin account](https://accounts.crowdin.com/register?continue=%2Fproject%2Fnvda).
+1. Message the [translators mailing list](https://groups.io/g/nvda-translations) or <info@nvaccess.org> to request being added as a translator.
+Please include your Crowdin username and the languages you wish to translate.
+
+### Setup PoEdit
+
+It is recommended that you use the latest version of PoEdit and NVDA for translating.
+Alternatively, you can use the [Crowdin web interface directly(https://support.crowdin.com/online-editor/).
+As PoEdit only supports viewing approved strings, large translators team need to co-ordinate submitting unapproved strings to prevent conflicts.
+Using Crowdin's interface avoids this problem.
+
+PoEdit supports connecting with Crowdin directly.
+PoEdit's homepage is: <http://www.poedit.net/>
+
+1. Download the latest Windows PoEdit version at <https://poedit.net/download>
+1. Install it by following the on-screen instructions, the default options should be sufficient.
+1. When launching PoEdit:
+    1. Choose "Translate cloud project"
+    1. Connect your Crowdin account
+    1. Select NVDA and the language you wish to translate
+
+### Translation reviews
+
+Translated strings will need to be reviewed and approved by a proofreader before being included in NVDA.
+A proofreader is required for each language.
+Proofreader status is granted on a case-by-case basis by messaging the [translators mailing list](https://groups.io/g/nvda-translations) or <info@nvaccess.org>
+
+Proofreaders approve strings using the [Crowdin web interface](https://support.crowdin.com/online-editor/).
+PoEdit only supports viewing approved strings.

```

Comment:
To be checked but, it seems to me, more precisely, that when you use synchronization of poedit with Crowdin, you get the approved strings from Crowdin, but of course, your personal changes (suggestions) also appear in poedit.

Reply:
```suggestion
PoEdit does not support viewing unapproved strings from other translators.
```